docs/docs/getting-started/docker.md

```diff
@@ -0,0 +1,19 @@
+# Development with Docker
+
+An altenative to installing Exograph directly on your machine is to use Docker. Exograph provides an image suitable for development that includes all Exograph and Postgres binaries.
+
+The development environment can be started using the following command:
+
+```sh
+# shell-command-next-line
+docker run --rm --platform linux/x86_64 \
```

Comment:
https://github.com/exograph/exograph/issues/1282#issuecomment-2393527035

I would suggest adding an example here not only for dev mode but for prod mode as well which includes building the intermediate representation

Reply:
Done. Please check https://github.com/exograph/exograph/pull/1283/commits/2eca5f51fb6c927b360e273895174112d867ef0d and let me know.

Reply:
I think this is great